04.ls/ls.rb

```diff
@@ -25,38 +25,34 @@ def main
 end
 
 def show_long_format(entries, path)
-  entries_long_format = []
+  long_format_entries = []
```

Comment:
行と列で構成される表と見なすとわかりやすく命名できるかもしれません。

```suggestion
  long_format_rows = []
```

![IMG_8916](https://github.com/NinomiyaW/ruby-practices/assets/1148320/a88f9460-f3db-4c66-ad68-4c6cf646da67)

Reply:
わかりやすく例を提示してくださりありがとうございます。